### PR TITLE
Fix Login Form

### PR DESCRIPTION
**Pull request by sunniraleigh**
- update login format
- ensure user password is authenticate using bcrypt gem

### CODE REVIEW

File: db/migrate/20220330211042_create_users.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ def change
     create_table :users do |t|
         # t.integer   :identifier, auto_increment: true, null: false
         t.string    :username
-        t.string    :password
+        t.string    :password_digest
```
Comment (epartakki) [inline db/migrate/20220330211042_create_users.rb:6]: What exactly is the difference in the change?

**Review by epartakki — APPROVED** [review]
thanks Sunni

**Review by amarrap — APPROVED** [review]
Awesome!!!!!!!!

**Review by luciebrock — APPROVED** [review]
looks good lad

**Review by kvangreen — APPROVED** [review]
Absolute genius

File: db/migrate/20220330211042_create_users.rb (hunk 1 above)
Comment (sunniraleigh) [reply db/migrate/20220330211042_create_users.rb:6]: This is for the bcrypt gem. While the rest of the app refers to the user's password as just `password`, to securely save it in the database the bcrypt gem calls it password_digest. I'm not entirely sure what it does but it makes the login w/ bcrypt work.

Comment (epartakki) [reply db/migrate/20220330211042_create_users.rb:6]: epic-- thank you!

**Pull request merged**